src/timetable/timetable.tsx

```diff
@@ -35,24 +35,27 @@ import { TimeTableProps, TimeTableRowData } from "./types";
 
 export const TimeTable = ({
     date,
-    emptyContent,
+    emptyContentMessage,
     rowData,
     loading,
     minTime = "00:00",
     maxTime = "23:00",
     minDate,
     maxDate,
     totalRecords,
+    showCurrentDateAsToday,
+    showDateAsShortForm,
     onPage,
     onRefresh,
     onNextDayClick,
     onPreviousDayClick,
+    onCalendarDateSelect,
     ...otherProps
 }: TimeTableProps): JSX.Element => {
     // =============================================================================
     // CONST, STATE, REF
     // =============================================================================
-    const testId = otherProps["data-testid"] || "timetable-container-id";
+    const testId = otherProps["data-testid"] || "container-testid";
```

Comment:
should destructure also testId as it'd get overridden by `{...otherProps}` anyway

```suggestion
    const testId = otherProps["data-testid"] || "timetable";
```

or actually can just let the consumer set the test id, can leave it undefined when not specified

Reply:
amended it so that the {...otherProps} comes first so that the testId takes precedence if thats ok!